Plugins/cmake-smoke-test/cmake-smoke-test.swift

```diff
@@ -166,29 +168,18 @@ enum OS {
 }
 
 extension Process {
-    func run() async throws {
```

Comment:
why?

Reply:
Process execution is hanging the CI action, haven't fully debugged it yet